app/models/concerns/waste_exemptions_engine/can_iterate_exemptions.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  module CanIterateExemptions
```

Comment:
I'm not quite understanding the name of this concern, would something like CanSortExemptions be a bit clearer?